cl/opinion_page/views.py

```diff
@@ -85,46 +90,89 @@
     RECAPDocument,
 )
 from cl.search.selectors import get_clusters_from_citation_str
-from cl.search.views import do_search
+from cl.search.views import do_es_search, do_search
 
 HYPERSCAN_TOKENIZER = HyperscanTokenizer(cache_dir=".hyperscan")
 
 
-async def court_homepage(request: HttpRequest, pk: str) -> HttpResponse:
+def court_homepage(request: HttpRequest, pk: str) -> HttpResponse:
     """Individual Court Home Pages"""
-    if pk not in ["tennworkcompcl", "tennworkcompapp", "me"]:
+
+    available_courts = [
+        "tennworkcompcl",
+        "tennworkcompapp",
+        "me",
+        "mo",
+        "moctapp",
+        "moctapped",
+        "moctappsd",
+        "moctappwd",
+        "miss",
+        "missctapp",
+    ]
+
+    if pk not in available_courts:
         raise Http404("Court pages only implemented for select courts.")
 
-    render_court = await Court.objects.aget(pk=pk)
+    render_court = Court.objects.get(pk=pk)
     render_dict = {
         "private": False,
         "pk": pk,
         "court": render_court.full_name,
     }
 
-    if "tennworkcomp" in pk:
+    if pk == "tennworkcompapp" or pk == "tennworkcompcl":
         courts = ["tennworkcompcl", "tennworkcompapp"]
         template = "tn-court.html"
     else:
         courts = [pk]
         template = "court.html"
 
+    court_seal = seal(pk, ImageSizes.SMALL)
+    if "moctapp" in pk:
+        # return mo seal
+        court_seal = seal("mo", ImageSizes.SMALL)
+    if "tennworkcomp" in pk:
+        # return tenn seal
+        court_seal = seal("tenn", ImageSizes.SMALL)
+
+    render_dict["court_seal"] = court_seal
+
     for court in courts:
         if "tennwork" in court:
             results = f"results_{court}"
         else:
             results = "results"
-        response = await sync_to_async(do_search)(
-            request.GET.copy(),
-            override_params={
-                "filed_after": (
-                    datetime.datetime.today() - datetime.timedelta(days=28)
```

Comment:
thanks for handling the elastic stuff but is there a reason we dropped sync to async for the not ES search

Reply:
I based it on another code in the repo, but async could be used, I already corrected it to use async